changelogs/fragments/5349-drop-gentoolkit-more-knobs.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - portage - use Portage's python module instead of calling gentoolkit-provided program in shell (https://github.com/ansible-collections/community.general/pull/5349)
+  - portage - add knobs for Portage's --backtrack and --with-bdeps options (https://github.com/ansible-collections/community.general/pull/5349)
```

Comment:
```suggestion
  - portage - use Portage's python module instead of calling gentoolkit-provided program in shell (https://github.com/ansible-collections/community.general/pull/5349).
  - portage - add knobs for Portage's ``--backtrack`` and ``--with-bdeps`` options (https://github.com/ansible-collections/community.general/pull/5349).
```

Reply:
You are still missing the trailing dots in both lines.

Reply:
Sorry! Missed those, there now.